scripts/gdb/lvglgdb/value.py

```diff
@@ -0,0 +1,31 @@
+import gdb
+
+from typing import Optional
+
+
+class Value(gdb.Value):
+    def __init__(self, value: "gdb.Value | Value"):
+        super().__init__(value)
+
+    def __getitem__(self, key):
+        try:
+            return Value(super().__getitem__(key))
+        except gdb.error:
+            return Value(super().__getattr__(key))
+
+    def __getattr__(self, key):
+        if hasattr(super(), key):
+            return Value(super().__getattribute__(key))
+        return Value(super().__getitem__(key))
+
+    def __setattr__(self, key, value):
```

Comment:
Why will we need set attribute?

Reply:
In runtime, we need

Reply:
Do you have an example? Normally we would only read the memory.